### PR TITLE
fix: slightly increase color contrast of gray-600 to meet 4.5:1 a11y standard on white backgrounds

### PR DESCRIPTION
**Pull request by jennifer-shehane**
The gray-600 color does not meet the a11y standard of 4.5:1 color contrast ratio on white backgrounds. This updates the color slightly to get it there so that this can be used as a back light gray color for designs within Cypress that meets the standard. 

[Example of violation within Cypress App.](https://cloud.cypress.io/projects/ypt4pf/runs/55275/accessibility/025590e09a6edf822a6d05538ae6d876f18ab8fb?impact=critical%2Cserious%2Cmoderate%2Cminor&ruleset=wcag21a%2Cwcag21aa%2Cbest-practice&snapshotId=ca59ca1dd184ed1f727624ea07042132b53ca648&violationId=color-contrast&targetId=Lm1iLVxbMTZweFxdLnRleHQtZ3JheS02MDA%3D)

### Before
Screenshot 2024-05-01 at 2 26 38 PM

### After

Screenshot 2024-05-01 at 2 27 20 PM

### Before
Screenshot 2024-05-01 at 2 23 11 PM

### After
Screenshot 2024-05-01 at 2 22 52 PM

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -7,5 +7,9 @@
     "editor.defaultFormatter": "esbenp.prettier-vscode"
   },
   "eslint.validate": ["javascript", "typescript", "vue"],
-  "cSpell.words": ["clsx", "docmenu"]
+  "cSpell.words": [
```
Comment (jennifer-shehane) [inline .vscode/settings.json:10]: not relevant, just my auto linting I guess 🙄

**Review by elevatebart — APPROVED** [review]

**Review by ryanjwilke — CHANGES_REQUESTED** [review]
I would caution against fixing our color contrast issues by changing the color. I mentioned this is Slack, but the new standard the team has shifted to is to just use `gray-700` in our designs instead. We used `gray-600` sometimes in the past before we paid too much attention to a11y requirements, but we shifted to using `gray-700` in most scenarios about a year ago.

**Comment by jennifer-shehane** [comment]
@ryanjwilke It is a slightly bigger shift in color to gray-700 than the slight change here. But I'm ok updating to gray-700 for these uses if that's the suggestion. Honestly we have a lot more uses of gray-500 on white backgrounds in the App, so I'm not sure the approach with those either.

#### Today's gray-600

Screenshot 2024-05-02 at 11 45 33 AM

#### gray-600 in this PR

Screenshot 2024-05-02 at 11 46 14 AM

#### gray-700

Screenshot 2024-05-02 at 11 45 49 AM

**Comment by ryanjwilke** [comment]
@jennifer-shehane Yep, I figured this might be for resolving a text contrast issue. I would suggest to use `gray-700` instead as that aligns with how the design team has been resolving those issues throughout all of our designs at this point.

**Pull request closed without merging**